src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/private_link_scope/network_security_perimeter_configuration/_list.py

```diff
@@ -0,0 +1,324 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "connectedmachine private-link-scope network-security-perimeter-configuration list",
```

Comment:
This command signature looks too long. Could you make it shorter and easier to input?

Reply:
The name is requested by a partner team so unfortunately we can't shorten it